src/Admin.js

```diff
@@ -1037,11 +1042,13 @@ class Admin {
                 this.views.$design_toolbox.empty();
                 this.views.$views_jsapp.empty();
 
-                let merge_view_update_toggle = '<div class="onoffswitch" style="display:none;"> <input type="checkbox" name="onoffswitch" '
-                    +'class="onoffswitch-checkbox" id="myonoffswitchmerge" onchange="liveUpdatesCheckboxChangeMerge(this);"> </input> <label class="onoffswitch-label" for="myonoffswitchmerge">'
-                    +'<span class="onoffswitch-inner"></span> <span class="onoffswitch-switch"></span> </label></div>';
+                let merge_view_update_toggle = this.templates.merge_view_update_toggle;
                 this.views.$views_jsapp.append(merge_view_update_toggle);
 
+                $('.onoffswitch-checkbox').on('change',(e)=>{
+                  this.liveUpdatesCheckboxChangeMerge(e,e.target);
```

Comment:
1. Keep tabs consistent - if it's 4 spaces throughout the file, it shouldn't be 2 spaces here.
2. Please separate args with a space.